doc/userguide/src/CreatingTestData/CreatingTestCases.rst

```diff
@@ -1037,34 +917,17 @@ easier to see what they test, but having potentially large number of
 these tests may mess-up statistics. Which style to use depends on the
 context and personal preferences.
 
-.. table:: Data-driven test with multiple data variations
-   :class: example
-
-   +-------------------+---------------+-------------------+---------+
-   |     Test Case     |   User Name   |      Password     |         |
-   +===================+===============+===================+=========+
-   | Invalid Password  | [Template]    | Login with invalid|         |
-   |                   |               | credentials should|         |
-   |                   |               | fail              |         |
-   +-------------------+---------------+-------------------+---------+
-   |                   | invalid       | ${VALID PASSWORD} |         |
-   +-------------------+---------------+-------------------+---------+
-   |                   | ${VALID USER} | invalid           |         |
-   +-------------------+---------------+-------------------+---------+
-   |                   | invalid       | whatever          |         |
-   +-------------------+---------------+-------------------+---------+
-   |                   | ${EMPTY}      | ${VALID PASSWORD} |         |
-   +-------------------+---------------+-------------------+---------+
-   |                   | ${VALID USER} | ${EMPTY}          |         |
-   +-------------------+---------------+-------------------+---------+
-   |                   | ${EMPTY}      | ${EMPTY}          |         |
-   +-------------------+---------------+-------------------+---------+
-
-.. tip:: In both of the above examples, column headers have been
-         changed to match the data. This is possible because on the
-         first row other cells except the first one `are ignored`__.
```

Comment:
This is a good tip and should be preserved. Headers should also be added at least to the first example.